### PR TITLE
Add sphinxcontrib-spelling

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by sannykr** [comment]
@jakirkham : Could you please review and merge.

File: recipes/sphinxcontrib-spelling/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (isuruf) [inline recipes/sphinxcontrib-spelling/meta.yaml]: Can you try `noarch: python` ?
```
build:
   noarch: python
```

File: recipes/sphinxcontrib-spelling/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  description: |
+    This package contains sphinxcontrb.spelling, a spelling checker 
+    for Sphinx-based documentation. It uses PyEnchant to produce 
+    a report showing misspelled words.
+  doc_url: https://bitbucket.org/dhellmann/sphinxcontrib-spelling
+  dev_url: https://bitbucket.org/dhellmann/sphinxcontrib-spelling
+
+extra:
+  recipe-maintainers:
+    - cshaley
```
Comment (isuruf) [inline recipes/sphinxcontrib-spelling/meta.yaml]: @cshaley, are you ok being a maintainer?

Comment (cshaley) [reply recipes/sphinxcontrib-spelling/meta.yaml]: Yes

**Comment by isuruf** [comment]
This is a noarch package. So CI for other platforms doesn't need to be run.

**Pull request merged**